wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -63,6 +63,42 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     }
   }
 
+  override def sendToAddresses(
```

Comment:
nit: Maybe make `sendToOutputs()` that we just delegate to here? I think we should probably keep this, but it might be nice to just use outputs too.